contentcuration/contentcuration/settings.py

```diff
@@ -86,6 +86,7 @@
     'webpack_loader',
     'django_filters',
     'mathfilters',
+    'django.contrib.postgres',
```

Comment:
Was this needed for the full text search support?

Reply:
Yes, sir. This enables use of SearchVector and friends.